scripts/acbuild-rkt-builder.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -ex
+
+MODIFY=${MODIFY:-""}
+FLAGS=${FLAGS:-""}
+IMG=${IMG:-"debian"}
+IMGVERSION=${VERSION:-"sid"}
+DOCKERIMG="$IMG:$IMGVERSION"
+ACIFILE=${ACIFILE:-"$IMG-$IMGVERSION.aci"}
+OUTACI="rkt-builder.aci"
+BUILDDIR=/opt/build-rkt
+SRC_DIR=/opt/rkt
+ACI_GOPATH=/go
+VERSION=${VERSION:-"v0.10.0"}
```

Comment:
I did want to do that, but it makes using it via rkt a pain because the image name becomes `coreos.com/rkt/builder:v0.10.0+git123abc-dirty` and then my `build-rir.sh` needs to guess at what image to use basically by looking at `rkt image list` so I opted out on attempting that.